.gitlab-ci.yml

```diff
@@ -833,7 +803,6 @@ iwyu:
     - .use_gko-cuda101-openmpi-gnu8-llvm7-intel2019
   variables:
     BUILD_OMP: "ON"
-    BUILD_CUDA: "ON"
     BUILD_CUDA: "HIP"
```

Comment:
I don't understand this setting?

Reply:
I thought it had something to do with hip for cuda, but I guess that's not the case. Anyway, the first option would be overridden by the second one, so I will set it just to the first one.